src/acceptance.spec.tsx

```diff
@@ -0,0 +1,107 @@
+import { runCucumber, SupportCode } from '@cucumber/fake-cucumber'
+import { GherkinStreams } from '@cucumber/gherkin-streams'
+import { Query as GherkinQuery } from '@cucumber/gherkin-utils'
+import { NdjsonToMessageStream } from '@cucumber/message-streams'
+import { IdGenerator } from '@cucumber/messages'
+import * as messages from '@cucumber/messages'
+import { Query as CucumberQuery } from '@cucumber/query'
+import assert from 'assert'
+import fs from 'fs'
+import glob from 'glob'
+import path from 'path'
+import React from 'react'
+import { pipeline, Writable } from 'stream'
+import { promisify } from 'util'
+
+import { CucumberQueryStream } from '../test-utils'
+import { render } from '../test-utils'
+import { EnvelopesQuery } from './EnvelopesQueryContext'
+import { components } from './index'
+
+describe('acceptance tests', () => {
+  jest.setTimeout(30000)
+
+  describe('with test data', () => {
+    const dir = __dirname + '/../testdata/good'
+    const files = fs.readdirSync(dir)
+
+    for (const file of files) {
+      if (file.match(/\.feature$/)) {
+        it(`can render ${file}`, async () => {
```

Comment:
I see no assertion in there.
I guess the test is failing if there is an error while rendering the component?

Reply:
> I guess the test is failing if there is an error while rendering the component?

Yeah I think that's the idea.

> Would it be possible to still have an assertion?

We could probably add a check to make sure the rendered element isn't empty, let me try that.

Reply:
Done!